README.md

```diff
@@ -1,6 +1,27 @@
-The repository contains specification to manipulate General Transit
-Feed Specification (GTFS) and GTFS Realtime:
-* [GTFS](/gtfs/README.md)
-* [GTFS Realtime](/gtfs-realtime/README.md)
+# General Transit Feed Specification (GTFS) Overview
+The General Transit Feed Specification (GTFS) is an [Open Standard](https://www.interoperablemobility.org/definitions/#open_standard) used to distribute relevant information about transit systems to riders. It allows public transit agencies to publish their transit data in a format that can be consumed by a wide variety of software applications. Today, the GTFS data format is used by thousands of public transport providers.GTFS consists of two formats: GTFS Schedule and GTFS Realtime.
 
-Please visit https://gtfs.org for information.
+[Read more about GTFS](https://gtfs.org/)
+
+# Get Monthly Updates on GTFS Developments
+The GTFS Digest is a resource generated monthly by [MobilityData](https://mobilitydata.org/) providing an overview of developments on GTFS. 
```

Comment:
I find this paragraph a bit unclear/vague. 
The broad "GTFS Developments" without anything else could be about GTFS tools or market insights, for example, whereas here this is more for folks interested in participating in the community.

I'd suggest something like this:

> The GTFS Digest is a monthly newsletter by [MobilityData](https://mobilitydata.org/) providing an overview of GTFS developments, discussions within our community, and ways you can get involved in shaping the future of GTFS.